resources/resources.md

```diff
@@ -0,0 +1,21 @@
+# Resources
+
+## Line Notes
+
+This indicates where there may be a typo or bug in the text, and is very helpful to the leaders as it shows them exactly where they have gone wrong.  There are three steps to achieve this:
```

Comment:
Can you wrap this at 80 characters? We also generally only use one space between sentences.